File: forum-service/src/main/java/com/ironhack/forumservice/model/Post.java
package com.ironhack.forumservice.model;

import org.springframework.data.annotation.Id;
import org.springframework.data.mongodb.core.mapping.Document;

@Document(collection = "forum")
public class Post {
    @Id
    private Long id;
    private String title;
    private String name;
    private String comment;

    public Post() {
    }

    public Post(Long id, String title, String name, String comment) {
        this.id = id;
        this.title = title;
        this.name = name;
        this.comment = comment;
    }

    public Long getId() {
        return id;
    }

    public void setId(Long id) {
        this.id = id;
    }

    public String getTitle() {
        return title;
    }

    public void setTitle(String title) {
        this.title = title;
    }

    public String getName() {
        return name;
    }

    public void setName(String name) {
        this.name = name;
    }

    public String getComment() {
        return comment;
    }

    public void setComment(String comment) {
        this.comment = comment;
    }
}

File: recipes-app/recipes-angularapp/src/app/models/post/post.ts
export class Post {

    constructor(
        private _id: number,
        private _title: string,
        private _name: string,
        private _comment: string
    ){}

    public get comment(): string {
        return this._comment;
    }
    public set comment(value: string) {
        this._comment = value;
    }
    public get name(): string {
        return this._name;
    }
    public set name(value: string) {
        this._name = value;
    }
    public get title(): string {
        return this._title;
    }
    public set title(value: string) {
     this._title = value;
    }
    public get id(): number {
        return this._id;
    }
    public set id(value: number) {
        this._id = value;
    }
}
